services/cognito/app/resources/user_resource.rb

```diff
@@ -6,9 +6,9 @@ class UserResource < Cognito::ApplicationResource
   filter :primary_identifier
 
   filter :query, apply: lambda { |records, value, _options|
-    query_by_non_id_attrs = %w[primary_identifier first_name last_name phone_number email_address]
-                            .map { |field| "#{field} ILIKE :non_id_query" }
-                            .join(' OR ')
+    searchable_columns = ['primary_identifier', 'first_name', 'last_name', 'phone_number', 'email_address',
```

Comment:
starts to feel cleaner to just concat the searchable_columns in a 'search string' and then search against that search string. I think that we've reached the threshold of complexity to think of a nicer solution. any further changes that add to the complexity of this, should trigger the discussion.